cmd/vic-machine/configure/configure.go

```diff
@@ -85,6 +105,59 @@ func (c *Configure) Flags() []cli.Flag {
 			Destination: &c.upgrade,
 			Hidden:      true,
 		},
+		cli.BoolFlag{
+			Name:        "regenerate-certs",
+			Usage:       "Generate self-signed certs and enable them on this VCH automatically",
+			Destination: &c.regenerateCerts,
+		},
+		cli.StringFlag{
+			Name:        "tls-key",
+			Value:       "",
+			Usage:       "Virtual Container Host private key file (server certificate)",
+			Destination: &c.skey,
+		},
+		cli.StringFlag{
+			Name:        "tls-cert",
+			Value:       "",
+			Usage:       "Virtual Container Host x509 certificate file (server certificate)",
+			Destination: &c.scert,
+		},
+		cli.StringFlag{
+			Name:        "tls-cname",
+			Value:       "",
+			Usage:       "Common Name to use in generated CA certificate when requiring client certificate authentication",
+			Destination: &c.cname,
+		},
+		cli.StringFlag{
+			Name:        "cert-path",
+			Value:       "",
+			Usage:       "The path to check for existing certificates and in which to save generated certificates. Defaults to './<vch name>/'",
+			Destination: &c.certPath,
+		},
+		cli.BoolFlag{
```

Comment:
Although, like most of these options, the real world effect of what you're doing is not clear. All of these flags need a redesign, and it kills me that I'm cloning the `create` flags here for that reason. This whole interface for configuring certificates is debt and we're duplicating it in `configure`.